daprdocs/content/en/developing-applications/building-blocks/jobs/howto-schedule-and-handle-triggered-jobs.md

```diff
@@ -20,13 +20,107 @@ When you [run `dapr init` in either self-hosted mode or on Kubernetes]({{< ref i
 
 In your code, set up and schedule jobs within your application.
 
-{{< tabs "Go" >}}
+{{< tabs ".NET" "Go" >}}
+
+{{% codetab %}}
+
+<!-- .net -->
+
+The following .NET SDK code sample schedules the job named `prod-db-backup`. Job data is housed in a backup database (`"my-prod-db"`) and is schedueled with `ScheduleJobAsync` from the Dapr Jobs client.  This provides the `jobData` which includes:
+- The backup `Task` name
+- The backup tasks's `Metadata` including:
+  - The database name (`DBName`)
+  - The database location (`BackupLocation`)
+
+```csharp
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using Dapr.Jobs;
+using Dapr.Jobs.Extensions;
+using Dapr.Jobs.Models;
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Services.AddDaprJobsClient();
+var app = builder.Build();
+
+// In practice, this would likely be in another service, but for completeness, we'll put this sample in a single file
+var DoProdDbBackupHandler = (DbBackupJobPayload? payload) => {
+	//Do something
+};
+
+// Set a handler for incoming jobs
```

Comment:
Little confused on this comment. This is "configure a map of handlers that are then called" Trying to be clear where the call back from the jobs schedule happens, since that is in `JobsController` yes?

Reply:
Well, it's the problem with making condensed examples in that I wanted to have a feature-complete demo though in practice, the handler itself would likely be handled in an injected service. I was planning on updating the example to have a more feature-complete sample (and perhaps link to it from here) that shows a more real-world scenario.

Because the Go example in this documentation splits between the SDK, HTTP, and gRPC samples (with a blurb calling out SDKS again at the bottom), I was trying to follow the same format wherein the SDK favors DI-based Jobs client registration and ASP.NET Core mapping (but then have a one-file demo) and then put the HTTP version in the controllers.